src/utils/date.ts

```diff
@@ -1,7 +1,25 @@
+const now = new Date()
+
```

Comment:
Wouldn't this keep the value constant after the file is loaded? Is probably safer to move to getSecondsFromDate